packages/react-components/src/components/ReactionButton.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(reaction) */
+import {
+  ContextualMenuItemType,
+  DefaultPalette,
+  IButtonStyles,
+  IContextualMenuItem,
+  mergeStyles,
+  Theme,
+  useTheme
+} from '@fluentui/react';
+/* @conditional-compile-remove(reaction) */
+import React, { useState } from 'react';
+/* @conditional-compile-remove(reaction) */
+import { ControlBarButton, ControlBarButtonProps } from './ControlBarButton';
+/* @conditional-compile-remove(reaction) */
+import { _HighContrastAwareIcon } from './HighContrastAwareIcon';
+/* @conditional-compile-remove(reaction) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(reaction) */
+import { reactionEmoji } from './utils/videoTileStylesUtils';
+/* @conditional-compile-remove(reaction) */
+import { emojiStyle } from './styles/ReactionButton.styles';
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * Props for {@link ReactionButton}.
+ *
+ * @beta
+ */
+export interface ReactionButtonProps extends ControlBarButtonProps {
+  /**
+   * Optional strings to override in component
+   */
+  strings?: Partial<ReactionButtonStrings>;
+  /**
+   * Click event to send reaction to meeting
+   */
+  onReactionClicked: (emoji: string) => Promise<void>;
+}
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * Strings of {@link ReactionButton} that can be overridden.
+ *
+ * @beta
+ */
+export interface ReactionButtonStrings {
+  /** Label of the button. */
+  label: string;
+  /** * Tooltip content when the button is disabled. */
+  tooltipDisabledContent?: string;
+  /** Tooltip content when the button is enabled. */
+  tooltipContent?: string;
+}
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * A button to send reactions.
+ *
+ * Can be used with {@link ControlBar}.
+ *
+ * @beta
+ */
+export const ReactionButton = (props: ReactionButtonProps): JSX.Element => {
+  const localeStrings = useLocale().strings.reactionButton;
+  const strings = { ...localeStrings, ...props.strings };
+  const theme = useTheme();
+  const styles = reactionButtonStyles(theme);
+  const onRenderIcon = (): JSX.Element => <_HighContrastAwareIcon iconName="Emoji2" />;
```

Comment:
I think we should conform to how the other icons that we have are named. we should be calling this `reactionIcon` or something like that and register it with the other icons that we have